crates/ra_assists/src/handlers/fill_match_arms.rs

```diff
@@ -307,6 +345,103 @@ mod tests {
         );
     }
 
+    #[test]
```

Comment:
Currently, the assist panics on a single element tuple:

```rust
match (Some(22), ) {   
}
```

Might be a good idea to fix it :)

Reply:
This one was trickier than I thought. The panic comes from `make::tuple_pat`. I didn't dig much into it, but I think it is because a single element tuple doesn't get parsed as a `TuplePat`. 

For now I handle this case by marking the assist as non-applicable if `tuple_len < 2`. I hope that is acceptable for now.